sklearn/metrics/cluster/supervised.py

```diff
@@ -586,16 +575,12 @@ def mutual_info_score(labels_true, labels_pred, contingency=None,
     labels_pred : array, shape = [n_samples]
         A clustering of the data into disjoint subsets.
 
-    contingency: None or array, shape = [n_classes_true, n_classes_pred]
+    contingency: {None, array, sparse matrix},
+                shape = [n_classes_true, n_classes_pred]
```

Comment:
@amueller Is this the right level of indentation for the numpydoc sphinx stuff? I never remember.